service-app/internal/types/client.go

```diff
@@ -0,0 +1,22 @@
+package types
+
+type ScannedCaseRequest struct {
+	BatchID        string `json:"batchId"`
+	CaseType       string `json:"caseType"`
+	CourtReference string `json:"courtReference,omitempty"`
+	ReceiptDate    string `json:"receiptDate"`
+	CreatedDate    string `json:"createdDate"`
+}
+
+type ScannedCaseResponse struct {
+	UUID string `json:"uuid"`
```

Comment:
Sorry, spec typo on my part, this should be `uid`:
```suggestion
	UID string `json:"uid"`
```